src/System Application/App/Json/src/Json.Codeunit.al

```diff
@@ -0,0 +1,64 @@
+// ------------------------------------------------------------------------------------------------
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See License.txt in the project root for license information.
+// ------------------------------------------------------------------------------------------------
+
+namespace System.Json;
+
+codeunit 5460 Json
+{
+    Access = Public;
```

Comment:
The public API will need to have some tests.

Reply:
Tests added https://github.com/microsoft/BCApps/pull/716/commits/b7de2bc7d00ac8434edbe9a0e238440ef3ed10de
<img width="1434" alt="image" src="https://github.com/microsoft/BCApps/assets/29101017/44dd531d-9dc6-4f15-91e1-3fe14cf9f41c">